bundles/org.openhab.binding.souliss/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,935 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="souliss"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+
+	<bridge-type id="gateway">
+		<label>Gateway</label>
+		<description>Represents a Souliss Gateway.</description>
+
+		<config-description>
+			<parameter name="GATEWAY_IP_ADDRESS" type="text" required="true">
+				<label>IP or Host Name</label>
+				<description>Will be resolved by discovery if auto configured
+				</description>
+			</parameter>
+			<parameter name="GATEWAY_PORT_NUMBER" type="integer" required="true">
+				<label>Gateway Port</label>
+				<description>Default is 230 UDP.
+				</description>
+				<default>230</default>
+			</parameter>
+			<parameter name="PREFERRED_LOCAL_PORT_NUMBER" type="integer" required="true">
+				<label>Preferred Local Port Number</label>
+				<description>Default port is 0. It means that it is random.
+				</description>
+				<default>0</default>
+			</parameter>
+
+			<parameter name="PING_INTERVAL" type="integer" min="1" max="300">
+				<label>Ping interval</label>
+				<description>Interval in seconds to check for device presence.</description>
+				<default>30</default>
+			</parameter>
+			<parameter name="SUBSCRIBTION_INTERVAL" type="integer" min="1" max="60">
+				<label>Subscription interval</label>
+				<description>Interval in minutes to subcribe Souliss Gateway.
+				</description>
+				<default>2</default>
+			</parameter>
+			<parameter name="HEALTHY_INTERVAL" type="integer" min="1" max="60">
+				<label>Healthy interval</label>
+				<description>Interval in seconds to send nodes healthy.
+				</description>
+				<default>35</default>
+			</parameter>
+
+			<parameter name="SEND_INTERVAL" type="integer" min="5" max="5000">
+				<label>Send interval</label>
+				<description>Interval in milliseconds to get packet from binding queue and send it to Souliss. First packet is sent
+					immediately.
+				</description>
+				<default>1000</default>
+			</parameter>
+
+			<parameter name="TIMEOUT_TO_REQUEUE" type="integer" min="0" max="10000">
+				<label>Interval in milliseconds to requeue packet in queue if not yet executed</label>
+				<description></description>
+				<required>true</required>
+				<default>5000</default>
+			</parameter>
+
+			<parameter name="TIMEOUT_TO_REMOVE_PACKET" type="integer" min="0" max="60000">
+				<label>Interval in milliseconds to remove packet from queue if not yet executed </label>
+				<description></description>
+				<required>true</required>
+				<default>20000</default>
+			</parameter>
+
+			<parameter name="USER_INDEX" type="integer" required="true">
+				<label>User Index</label>
+				<description>Generally the default value work good. It must is different from other user interfaces (ex: SoulissApp)
```

Comment:
```suggestion
				<description>Generally the default value will work. It must be different from other user interfaces (ex: SoulissApp)
```

Reply:
Actually I think this text should be in the readme.md and in this place the description should be able what the parameter DOES. I have not read the readme today and by reading this I have no idea what it is for.

Reply:
fixed